### DISCUSSION TITLE
Ne translation language - Slovenian

### DISCUSSION THREAD

**Discussion by thehijacker** [Vito0912/abs_flutter#124] [General]
I am registered on Weblate and wish to add Slovenian translation. Language was added but I have no strings to translate. Can you check please? Thank you.

**Comment by thehijacker** [comment]
Weblate shows a message:

`No strings found!`

**Comment by Vito0912** [comment]
Should be working now

**Reply by thehijacker** [reply]
I can see the strings to translate now. Started with translation. Hope to have it finished today. Thank you.
